src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/FileIO/TextFieldParser.vb

```diff
@@ -0,0 +1,1576 @@
+        '''  An array of the strings that indicate a line is a comment
+        ''' </summary>
+        ''' <value>An array of comment indicators</value>
+        ''' <remarks>Returns an empty array if not set</remarks>
+#Disable Warning CA1819 ' Properties should not return arrays
+        <EditorBrowsable(EditorBrowsableState.Advanced)>
+        Public Property CommentTokens() As String()
+#Enable Warning CA1819 ' Properties should not return arrays
+            Get
+                Return m_CommentTokens
+            End Get
+            Set(ByVal value As String())
+                CheckCommentTokensForWhitespace(value)
+                m_CommentTokens = value
+                m_NeedPropertyCheck = True
+            End Set
+        End Property
+
+        ''' <summary>
+        '''  Indicates whether or not there is any data (non ignorable lines) left to read in the file
+        ''' </summary>
+        ''' <value>True if there's more data to read, otherwise False</value>
+        ''' <remarks>Ignores comments and blank lines</remarks>
+        Public ReadOnly Property EndOfData() As Boolean
+            Get
+                If m_EndOfData Then
+                    Return m_EndOfData
+                End If
+
+                ' Make sure we're not at end of file
+                If m_Reader Is Nothing Or m_Buffer Is Nothing Then
+                    m_EndOfData = True
+                    Return True
+                End If
+
+                'See if we can get a data line
+                If PeekNextDataLine() IsNot Nothing Then
+                    Return False
+                End If
+
+                m_EndOfData = True
+                Return True
+            End Get
+        End Property
+
+        ''' <summary>
+        '''  The line to the right of the cursor.
+        ''' </summary>
+        ''' <value>The number of the line</value>
+        ''' <remarks>LineNumber returns the location in the file and has nothing to do with rows or fields</remarks>
+        <EditorBrowsable(EditorBrowsableState.Advanced)>
+        Public ReadOnly Property LineNumber() As Long
+            Get
+                If m_LineNumber <> -1 Then
+
+                    ' See if we're at the end of file
+                    If m_Reader.Peek = -1 And m_Position = m_CharsRead Then
+                        CloseReader()
+                    End If
+                End If
+
+                Return m_LineNumber
+            End Get
+        End Property
+
+        ''' <summary>
+        '''  Returns the last malformed line if there is one.
+        ''' </summary>
+        ''' <value>The last malformed line</value>
+        Public ReadOnly Property ErrorLine() As String
+            Get
+                Return m_ErrorLine
+            End Get
+        End Property
+
+        ''' <summary>
+        '''  Returns the line number of last malformed line if there is one.
+        ''' </summary>
+        ''' <value>The last malformed line, line number</value>
```

Comment:
>line, line number [](start = 38, length = 17)

Perhaps just `line number`, which would match `<summary>` comment.

Reply:
Sorry, I meant the entire comment should be `<value>The last malformed line number</value>`.

---
In reply to: [227486116](https://github.com/dotnet/corefx/pull/32668#discussion_r227486116) [](ancestors = 227486116)